src/App.jsx

```diff
@@ -14,57 +15,52 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {good ? `${good} is selected` : 'No goods selected'}
+        {good && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            onClick={() => setGood('')}
+            className="delete ml-3"
+          />
+        )}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map(item => {
+            const isActive = good === item;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+            return (
+              <tr
+                data-cy="Good"
+                className={isActive ? 'has-background-success-light' : ''}
```

Comment:
Consider using keys while rendering lists

Reply:
@anastasiiavorobiova  Yes, I know, but here it's just an array, and I can't use Indexes for keys. I skipped this one for this task.
Or Should I modify the entry array?

Reply:
You can use `item` as a key. The item is unique and stable in this case

Reply:
@anastasiiavorobiova oh really? Thanks!